test/fixtures/bpmn/features/replace/cancel-events.bpmn

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" id="Definitions_1" targetNamespace="http://bpmn.io/schema/bpmn">
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" id="Definitions_1" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="4.3.0">
```

Comment:
Maybe we should squash both commits at all? The first commit is shipping the test case and the second one the actual commit, right (correct me if I'm wrong)?

Despite I like the test-driven approach, one single commit with the fix and the tests would keep our commit history even cleaner.

Reply:
> Maybe we should squash both commits at all? The first commit is shipping the test case and the second one the actual commit, right (correct me if I'm wrong)?

Right, my intention was to make it clearer for the reviewer , since the fix is so small and the test refactorings were big compared to that :-)

But yeah, I agree with the clean commit structure